### DISCUSSION TITLE
requestDevice only bluetooth printer options

### DISCUSSION THREAD

**Discussion by brandomcombr** [capacitor-community/bluetooth-le#612] [Q&A]
I'm using requestDevice but it opens a window with many bluetooth device options, I want to know if there's a way to show it in the bluetooth printer only options

**Comment by pwespi** [comment]
You can pass [`options`](https://github.com/capacitor-community/bluetooth-le#requestbledeviceoptions) to requestDevice to filter by service, name or namePrefix.

**Reply by mozhuilungdsuo** [reply]
@pwespi  is there any standard service uuid's for bluetooth printers ?
